code/style.css

```diff
@@ -1,7 +1,252 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+*, *:before, *:after {
+  box-sizing: inherit;
 }
+
+html {
+  box-sizing: border-box;
+}
+
+body, html {
+  font-family: 'Oxygen', sans-serif;
+  background-color: antiquewhite;
+  margin: 0;
+}
+  /* Hero-image with Header-text*/
+.hero-image {
+  display: flex;
+  flex-direction: column;
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("./tvättahåret.jpeg");
+  background-position: bottom;
+  background-size: cover;
+  background-repeat: no-repeat;
+  align-self: center;
+  background-attachment: fixed;
+  align-items: center;
+  justify-content: center;
+  height: 500px;
+}
+
+.logo {
+  width: 40%;
+}
+
+.hero-text {
+  text-align: center;
+  color: antiquewhite;
+  font-size: 90px;
+}
+
+.hero-header {
+  align-self: center;
+}
+
+/* Information */
+.information{
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-items: center;
+  background-color: rgb(16, 37, 54);
+}
+
+h2{
+  color: antiquewhite;
+}
+
+p {
+  display: flex;
+  width: 100%;
+  font-size: 14px;
+  text-align: center;
+  margin-bottom: 40px;
+  color: white;
+}
+
+.information button {
+  border: solid antiquewhite;
+  color: antiquewhite;
+  padding: 20px;
+  margin-bottom: 20px;
+}
+
+
+/* Booking-system */
+.booking-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 100%;
+}
+
+.moodbild {
+  width: 100%;
+}
+
+.booking{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: antiquewhite;
+  width: 100%;
+  margin: 20px;
+}
+
+label{
+  display: flex;
+  flex-direction: column;
+  padding: 5px;
+  margin: 5px;
+  font-size: 12px;
+}
+
+input{
+  margin-right: 10px;
+  border-radius: 15px;
+  border: solid rgb(16, 37, 54);
+}
```

Comment:
You have a `border-radius` which can create an interesting outline when you focus the input element.  You may have noticed, something what you see in the second input field in this screenshot.
![image](https://user-images.githubusercontent.com/807352/92369508-662d7700-f0f9-11ea-855a-a64e148e5b37.png)

If you want to retain that mechanism of focus, but also allow your outline to conform to your styling, you can try something like:
```css
input:focus {
    outline: none;
    box-shadow: 0 0 1pt 1pt black;  /* Modify this to suit your needs */
}
```
The first input element in the screenshot has this styling.